onboarding-checklist.md

```diff
@@ -0,0 +1,145 @@
+multiple Github repositories
+- [ ] Bookmark [VM JIRA] to work with the rest of VM Fusion.
+- [ ] In addition to your CISA email address `first.last@cisa.dhs.gov`, you'll
+also get an account on Trio, which is a Google-based account used for
+cross-team and cross-agency collaboration and is generally
+`first.last@trio.dhs.gov`. Trio may take a few days to get set up.
+  - Once you're set up on Trio, visit
+  [chat.google.com](https://chat.google.com) and join applicable rooms.
+- [ ] Make sure you have been added to several [dev team Google Groups] so you
+can participate in team-wide internal communication:
+  - [ ] [ALL Trio users@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/all)
+  - [ ] [CISA COOL](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisa-cool-group)
+  - [ ] [cisagov-github](https://groups.google.com/a/trio.dhs.gov/forum/#!forum/cisagov-github-group)
+  - [ ] [NCATS@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats)
+  - [ ] [NCATS AWS Support@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-aws-support)
+  - [ ] [NCATS Infrastructure Development@beta.dhs.gov](https://groups.google.com/a/beta.dhs.gov/d/forum/ncats-dev)
+- [ ] Add the [NCATS Google Drive folder] to your Google Drive. If you create
+or move a doc there, it'll get the right access permissions for team members
+to be able to view and edit it.
+
+## Role-specific items ##
+
+### Dev Team ###
+
+Useful links:
+
+- [ ] [NCATS Data Dictionary](https://github.com/cisagov/ncats-data-dictionary)
+repo for information about what's stored in the MongoDB database
+- [ ] [CISA COOL AWS Accounts](https://docs.google.com/spreadsheets/d/1It0FIlG7ZxTwrRV-zVYUAMw18L6ZstQiAYT7vGYf5VE/edit#gid=2069139012)
+sheet for AWS accounts reference
+- [ ] [Performance Plans](https://docs.google.com/spreadsheets/d/1UaGI8u70CXOGBtvJFQ85vCkPbqyDh5PVws0Xu893piw/edit?ts=5e558417&pli=1#gid=474573185)
+sheet, until performance plans are individualized again (sometime after
+June 2020)
+- [ ] [COOL Wiki](https://github.com/cisagov/cool-system/wiki/)
+- [ ] Set up your development environment on your CISA-provided Mac laptop
+(Link TBD)
+
+## Buddy checklist ##
+
+- [ ] Introduce yourself to the new team member and give them some of your
+background so they know who you are.
+- [ ] Identify a straightforward, well-groomed story in progress that involves
+their skills domain, schedule a meeting with the owner for an introduction (if
+it's not you), and set up pairing sessions several times in the first week on
+the project.
+- [ ] Identify a straightforward, well-groomed first story, ideally something
+they could conceivably complete in their first two/three weeks using their
+existing skills. Discuss the context with them, then make them the assignee.
+- [ ] Discuss suggestions for how the onboarding experience could have been
+improved and open as PRs on [the onboarding template]
```

Comment:
I think `[the onboarding template]` is a broken link.  Also this is a complete sentence.

Reply:
Yup, the link tag didn't match the links at the bottom any more. Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4